### DISCUSSION TITLE
State Parameters in Google OAuth does not reflect in Context.

### DISCUSSION THREAD

**Discussion by bamne123** [strapi/strapi#6371] [Legacy Topics]
Hi Guys,

**What I am doing?**
I m trying to pass State parameter to `Google OAuth 2.0`.

**What is Expected?**
I should get the State parameter passed into Context.

**What I m getting?**
When I execute I can See the Parameters in Url logged by Request Logger.
But Same values are not present in Context.

![Google-OAuth issue](https://user-images.githubusercontent.com/10872173/82988682-2c87a300-a017-11ea-8982-cfee65e69ee7.JPG)

**My Current Setup**
Strapi => 3.0.0-beta.20.3
Node => 12.16.2

**Comment by lauriejim** [comment]
Hello! Can you please give us more detail, it's quite hard to understand what you are trying to do.
